pyrsa/data/__init__.py

```diff
@@ -1,6 +1,9 @@
 from .dataset import Dataset
 from .dataset import load_dataset
 from .dataset import dataset_from_dict
+from .dataset import merge_subsets
+from .dataset import odd_even_split
```

Comment:
Would it perhaps make sense to have these available as static methods on the `DataSet` class? That way the user won't have to import so many functions

Reply:
As the merge_subsets function takes in lists of datasets, it's more useful as a function than as a method. I'll experiment with writing odd_even_split as a method.

Reply:
Okay, this should work now.